src/components/Login/DemoLogin.js

```diff
@@ -0,0 +1,186 @@
+import React, { useState, useEffect } from 'react';
+import { func, shape } from 'prop-types';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  ScrollView,
+} from 'react-native';
+import * as AppAuth from 'expo-app-auth';
+import Client from 'fhir-kit-client';
+
+// smartapp auth with provided patient
+const fhirIss = 'https://launch.smarthealthit.org/v/r4/sim/eyJrIjoiMSIsImIiOiI2ODk4OTJiZC1kY2JlLTQxZmMtODY1MS0zOGExZDA4OTM4NTQifQ/fhir';
+
+const initializeFhirClient = (baseUrl, accessToken) => {
+  if (!accessToken) {
+    return new Client({ baseUrl });
+  }
+  return new Client({
+    baseUrl,
+    customHeaders: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+  });
+};
+
+export async function signInAsync() {
+  const fhirClient = initializeFhirClient(fhirIss);
+  const { authorizeUrl, tokenUrl } = await fhirClient.smartAuthMetadata();
+
+  const config = {
+    serviceConfiguration: {
+      authorizationEndpoint: authorizeUrl.toString(),
+      tokenEndpoint: tokenUrl.toString(),
+    },
+    additionalParameters: {
+      aud: fhirIss,
+    },
+    clientId: 'example-client-id',
+    clientSecret: 'example-client-secret',
+    // redirectUrl:
+    //   Platform.OS === "android"
+    //     ? "com.reactnativeoauth:/callback"
+    //     : "org.reactjs.native.example.ReactNativeOauth:/callback",
+    scopes: [
+      'openid',
+      'fhirUser',
+      'patient/*.*',
+      'launch/patient',
+      'online_access',
+    ],
+  };
+
+  let result;
+  try {
+    result = await AppAuth.authAsync(config);
+    console.log('Auth result from signInAsync', result);
+  } catch (error) {
+    console.warn('error', error);
+  }
+
+  return result;
+}
+
+const DemoLogin = () => {
```

Comment:
it's no longer a Demo

Reply:
Yeah that true. I've refactored the name.